src/AutomatedCar/SystemComponents/DummySensor.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace AutomatedCar.SystemComponents
+{
+    using System;
+    using AutomatedCar.Models;
+    using AutomatedCar.SystemComponents.Packets;
+    public class DummySensor : SystemComponent
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DummySensor"/> class.
+        /// </summary>
+        /// <param name="virtualFunctionBus">R.</param>
+        public DummySensor(VirtualFunctionBus virtualfunctionbus) : base(virtualfunctionbus)
+        {
+            this.virtualFunctionBus = this.virtualFunctionBus;
```

Comment:
A dummy sensor szerintem törölhető, de ez a sor ez szerintem `this.virtualFunctionBus = virtualFunctionBus;` akar lenni, és amúgy fölösleges, mert a SystemComponent elvégzi ugyan ezt